### PR TITLE
Adding support for collection views

### PR DESCRIPTION
**Pull request by Renatdz**
Hey Juanpe, good afternoon!

I'm in love with your framework, but him doesn't support CollectionView, so I made It for you. 

Can you check this pull request? 

You can erase the example classes, I was only testing. 

Best regards,

Renato Mendes
Sao Paulo/Brazil

### CODE REVIEW

File: Sources/Collections/SkeletonUITableViewDataSource.swift [hunk 1]
```diff
@@ -17,6 +17,7 @@ public protocol SkeletonTableViewDataSource: UITableViewDataSource {
 public extension SkeletonTableViewDataSource {
     
     func collectionSkeletonView(_ skeletonView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        print(Int(ceil(skeletonView.frame.height/skeletonView.rowHeight)))
```
Comment (olivier-carbon) [inline Sources/Collections/SkeletonUITableViewDataSource.swift]: This should probably be removed

File: Sources/Collections/SkeletonUICollectionViewDataSource.swift [hunk 2]
```diff
@@ -8,4 +8,19 @@
...
+    func numSections(in collectionSkeletonView: UICollectionView) -> Int
+    func collectionSkeletonView(_ skeletonView: UICollectionView, numberOfItemsInSection section: Int) -> Int
+    func collectionSkeletonView(_ skeletonView: UICollectionView, cellIdenfierForItemAt indexPath: IndexPath) -> ReusableCellIdentifier
+}
+
+public extension SkeletonUICollectionViewDataSource {
+    
+    func collectionSkeletonView(_ skeletonView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        guard let flowlayout = skeletonView.collectionViewLayout as? UICollectionViewFlowLayout else { return 0 }
+        print(Int(ceil(skeletonView.frame.height/flowlayout.itemSize.height)))
```
Comment (olivier-carbon) [inline Sources/Collections/SkeletonUICollectionViewDataSource.swift]: Print statement not necessary.

File: Sources/Collections/SkeletonUITableViewDataSource.swift (hunk 1 above)
Comment (Renatdz) [reply Sources/Collections/SkeletonUITableViewDataSource.swift]: Fixed on e3e8d5b

File: Sources/Collections/SkeletonUICollectionViewDataSource.swift (hunk 2 above)
Comment (Renatdz) [reply Sources/Collections/SkeletonUICollectionViewDataSource.swift]: Fixed on e3e8d5b

**Comment by alessioarsuffi** [comment]
Hi guys, any update on this? I'm looking forward to use Skeleton inside UICollectionViews, thanks 👍

**Comment by Renatdz** [comment]
Hi @alessioarsuffi, I forgot to update the pull request.

**Comment by Juanpe** [comment]
Hi @Renatdz, thanks for your request! great work! 👍 
I'm going to check it and I'm going to update de README file to include this feature.

**Comment by alessioarsuffi** [comment]
Thanks you all 💯

**Comment by Renatdz** [comment]
Thank you guys!

**Pull request merged**